monitoring_versioned_docs/version-22.10/administration/parameters/data.md

```diff
@@ -0,0 +1,45 @@
+---
+id: data
+title: Data
+---
+
+### Graphs rebuild
+
+When, for example after a database restore, you need to retrieve performance and graph data, go to the **Administration -> Parameters -> Data** menu.
```

Comment:
sostituire la frae con il seguente:

When you need to restore a RRD graphs because it's maybe corrupt or you have done a database restore go to
**Administration -> Parameters -> Data**

Reply:
Ho modificatola frase come descritto e ho aggiornato il file data.md